conf/application.conf

```diff
@@ -34,14 +34,14 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.i18n.langCookieHttpOnly: "true"
 
 # Change this value to true to enable Welsh translations to be loaded from messages.cy, and to display the language toggle
-features.welsh-language-support = false
+features.welsh-language-support = true
 
 accessibility-statement.service-path = "/gvms-service-availability"
 
 # Replace play.i18n.langs with the commented out line below when your service has been fully translated into Welsh
 # to enable Welsh translations for all content, including the standard headers and footers.
 # play.i18n.langs = ["en", "cy"]
-play.i18n.langs = ["en-GB"]
+play.i18n.langs = ["en-GB", "cy"]
```

Comment:
minor: we can remove above commented line 43

Reply:
updated - thanks @sureshhmrc